apps/cross-components/src/global.scss

```diff
@@ -0,0 +1,3 @@
+// empty
```

Comment:
Script fail if file is empty, but may be better will be to just remove the how file.